packages/liferay-npm-scripts/README.md

```diff
@@ -63,6 +63,22 @@ For example
 liferay-npm-scripts test --watch
 ```
 
+### theme
+
+```sh
+liferay-npm-scripts theme TASK
+```
+
+Inside a theme directory, runs the Gulp task, `TASK`, from [liferay-theme-tasks](https://github.com/liferay/liferay-js-themes-toolkit/tree/master/packages/liferay-theme-tasks), automatically passing settings for use inside [liferay-portal](https://github.com/liferay/liferay-portal).
```

Comment:
I think it might be interesting to link to the list of available tasks:

```suggestion
Inside a theme directory, runs one of the [available Gulp tasks](https://github.com/liferay/liferay-js-themes-toolkit/tree/master/packages/liferay-theme-tasks#available-tasks), `TASK`, from [liferay-theme-tasks](https://github.com/liferay/liferay-js-themes-toolkit/tree/master/packages/liferay-theme-tasks), automatically passing settings for use inside [liferay-portal](https://github.com/liferay/liferay-portal).
```

Reply:
Sounds good. I also though about mentioning that you can run `theme help` to see all available tasks (avoiding the need to link to documentation that might get out of sync), but many of them are so low-level and "implementation-detail-ish" that I prefer not to advertise them. Your idea is better.